backend/src/index.js

```diff
@@ -104,10 +99,12 @@ io.on( 'connection', socket => {
 			return;
 		}
 
-		issueUsers.delete( socket.session.login );
+		delete issueUsers[ socket.session.login ];
 
 		const users = getUsers( issueUsers );
 
+		await storeUsers( issueKey, JSON.stringify( issueUsers ) );
```

Comment:
The `removeUser` function is still invalid. If the socket was the last session socket we still need to update the database